packages/atlas/src/components/_video/VideoHero/VideoHero.styles.ts

```diff
@@ -92,3 +93,7 @@ export const ButtonsContainer = styled.div`
     flex-shrink: 0;
   }
 `
+
+export const TitleText = styled(Text)<IsCategoryProp>`
+  ${({ isCategory }) => (isCategory ? `overflow-y: auto` : ``)};
```

Comment:
Now category variant has no limit
```suggestion
  ${({ isCategory }) => (!isCategory ? `overflow-y: auto` : ``)};
```

Reply:
@lcoenengithub You didn't apply my suggestion, video hero on home is still limited
![Zrzut ekranu 2022-02-9 o 11 52 09](https://user-images.githubusercontent.com/13706246/153183702-023f51e7-4354-48e3-b7c2-fd5610bd74da.png)

Reply:
Oops, my bad.